src/screens/ImageViewer/index.js

```diff
@@ -8,6 +8,8 @@ const ImageViewerScreen = ({route, navigation}) => {
     headerTitle: title
   });
 
-  return <ImageViewer imageUrls={images} index={index} />;
+  return (
+    <ImageViewer imageUrls={images} index={index} loadingRender={() => <ActivityIndicator />} />
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `loadingRender={() => <ActivityIndicator />}` in the `ImageViewer` component is a good practice. It provides visual feedback to the user when the images are loading, improving the overall user experience.
```diff
- return <ImageViewer imageUrls={images} index={index} />;
+ return (
+   <ImageViewer imageUrls={images} index={index} loadingRender={() => <ActivityIndicator />} />
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->